shuffle/cli/src/new.rs

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use diem_genesis_tool::validator_builder::ValidatorConfig;
+use diem_types::{account_address::AccountAddress, on_chain_config::VMPublishingOption};
+use move_cli::{
+    sandbox,
+    sandbox::utils::{on_disk_state_view::OnDiskStateView, Mode, ModeType},
+};
+use move_lang::shared::AddressBytes;
+use once_cell::sync::Lazy;
+use serde::{Deserialize, Serialize};
+use std::{
+    collections::{BTreeMap, HashMap},
+    fs,
+    path::{Path, PathBuf},
+};
+use toml;
+
+/// Default blockchain configuration
+pub const DEFAULT_BLOCKCHAIN: &str = "goodday";
+
+pub fn handle(blockchain: String, pathbuf: PathBuf) -> Result<()> {
+    let project_path = pathbuf.as_path();
+    println!("Creating shuffle project in {}", project_path.display());
```

Comment:
nit: can we also print name of project here

Reply:
Will do in follow up PR